src/dialogs/settingsdialog.ui

```diff
@@ -979,6 +979,7 @@
                    <font>
                     <weight>50</weight>
                     <bold>false</bold>
+                    <kerning>true</kerning>
```

Comment:
Did you add this on purpose?

Reply:
No, I think it is because my system has these things as defaults? Or some other reason maybe.

Reply:
Then please remove it as well. 😁